lib/api/src/Cardano/Wallet/Api/Types/Error.hs

```diff
@@ -293,6 +301,34 @@ data ApiErrorTxOutputLovelaceInsufficient = ApiErrorTxOutputLovelaceInsufficient
         via DefaultRecord ApiErrorTxOutputLovelaceInsufficient
     deriving anyclass NFData
 
+data ApiErrorOutputTokenQuantityExceedsLimit = ApiErrorOutputTokenQuantityExceedsLimit
+    { address
+        :: !Text
+    , policyId
+        :: !Text
```

Comment:
Request: use `TokenPolicyId` here.

```suggestion
        :: !(ApiT TokenPolicyId)
```

Reply:
done

Reply:
This seems in conflict with the idea that `ApiT` is only used "outside" . But ok, the wallet is going in one direction the deposit wallet in another :whatever:

Reply:
> This seems in conflict with the idea that `ApiT` is only used "outside" . But ok, the wallet is going in one direction the deposit wallet in another :whatever:

Hi @paolino! I'm curious, what solution would you suggest here?